tests/e2e/tests/utils.py

```diff
@@ -14,7 +14,7 @@ class RelengHelper:
         'FIREFOX_AURORA': 'firefox-aurora-latest',
         'FIREFOX_ESR_NEXT': None,  # no checks to run on this product
```

Comment:
flake8 runtests: commands[0] | flake8 .
./tests/utils.py:15:9: F601 dictionary key 'FIREFOX_ESR_NEXT' repeated with different values
./tests/utils.py:17:9: F601 dictionary key 'FIREFOX_ESR_NEXT' repeated with different values

This is because you've repeated it again by changing the FIREFOX_ESR define, here.